server/tokens.go

```diff
@@ -86,7 +87,13 @@ func (c *Server) BlindedTokenIssuerHandlerV2(w http.ResponseWriter, r *http.Requ
 			return appErr
 		}
 
-		signedTokens, proof, err := btd.ApproveTokens(request.BlindedTokens, issuer.SigningKey)
+		// get latest signing key from issuer
+		var signingKey *crypto.SigningKey
+		if len(issuer.Keys) > 0 {
+			signingKey = issuer.Keys[len(issuer.Keys)-1].SigningKey
+		}
+
+		signedTokens, proof, err := btd.ApproveTokens(request.BlindedTokens, signingKey)
```

Comment:
The code above issuer.Keys > 0 suggests that issuer.Keys can be 0. If so is there a potential for nil being passed to ApproveTokens. I think the same goes for other checks like these in the handlers